tinyquery/runtime.py

```diff
@@ -558,6 +558,24 @@ def _evaluate(self, num_rows, column):
                               values=values)
 
 
+class LastFunction(AggregateFunction):
+    def check_types(self, rep_list_type):
+        return rep_list_type
+
+    def _evaluate(self, num_rows, column):
+        values = []
+        if len(column.values) == 0:
+            values = [None]
+
+        if column.mode == tq_modes.REPEATED:
+            values = [repeated_row[-1] if len(repeated_row) > 0 else None
```

Comment:
Also, looking at this code it looks like the returned value can have NULL's in it if the row is empty.  Does this match what bigquery does?  The documentation doesn't seem to say.

It would be great if you could have a test-case for that as well.